pyqtorch/circuit.py

```diff
@@ -215,6 +215,112 @@ def sample(probs: Tensor) -> Counter:
             return list(map(lambda p: sample(p), probs))
 
 
+class DropoutQuantumCircuit(QuantumCircuit):
+    def __init__(
+        self,
+        n_qubits: int,
+        operations: list[Module],
+        dropout_mode: DropoutMode = DropoutMode.ROTATIONAL,  # type: ignore
```

Comment:
Ideally, after looking into the StringEnum you would be able to unmute mypy type checks.